### ISSUE TITLE
Accessibility issue on DropdownList 

### ISSUE THREAD

**Issue by bmdoherty** [jquense/react-widgets#1112]
Many thanks for this library its much appreciated!

Whilst migrating from v4 to v5 I noticed what seems to be an issue with the DropdownList component's accessibility. 
If filter prop is not present  screen readers do not announce the keyboard navigation between options, instead it seems to announce the text input. Setting `filter={false}`  we get the screen reader announcement back. Attaching 2 screenshots that hopefully describe the issue

![2021-12-02 (4)](https://user-images.githubusercontent.com/1644770/144464280-ada3e1ec-a2bd-4268-b16b-ace4e3072123.png)

![2021-12-02 (3) (1)](https://user-images.githubusercontent.com/1644770/144464444-d36ed927-f921-4982-b4ec-6861988f654c.png)

**Comment by bmdoherty** [comment]
thanks for quick reply, It's ok for my current project, as we never had filter before so I don't think they will miss it, so we are ok with `filter={false}`. Just thought I mention it, as it took me a while to work out
